std/experimental/logger/filelogger.d

```diff
@@ -200,3 +227,20 @@ class FileLogger : Logger
     assert(tl !is null);
     stdThreadLocalLog.logLevel = LogLevel.all;
 }
+
+@safe unittest
+{
+    import std.file : exists, rmdirRecurse;
+
+    string logpath = "super/long/path/";
+    string logname = logpath ~ "output.log";
+
+    assert(!exists(logpath));
+    auto fl = new FileLogger(logname, LogLevel.all, CreateFolder.yes);
+    fl.log("Hello World");
+    assert(exists(logpath));
+
+    fl.file.close();
+    () @trusted { rmdirRecurse(logpath); }();
```

Comment:
How about `scope(exit)`  in the beginning, s.t. the temporary folder is removed even in case of failures?